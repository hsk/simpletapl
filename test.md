# ここのところやってたこと

- PrologにTAPLのOCamlコードを移植。
- 移植したコードを更に短くした。
- 本質的な部分の取り出しを行った。
