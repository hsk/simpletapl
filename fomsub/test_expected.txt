lambda X.lambda x:X.x : All X.X -> X
lambda x:All X.X -> X.x : (All X.X -> X) -> (All X.X -> X)
lambda x:Top.x : Top -> Top
lambda x:Top.x : Top
lambda x:Top.x : Top -> Top
lambda X<:Top -> Top.lambda x:X.x x : All X <: Top -> Top.X -> Top
